k9mail-library/src/main/java/com/fsck/k9/mail/Part.java

```diff
@@ -7,6 +7,8 @@
 public interface Part {
     void addHeader(String name, String value) throws MessagingException;
 
+    void addRawHeader(String name, String raw) throws MessagingException;
```

Comment:
I'm not quite satisfied with this part of the code. Chances are we'll get rid of the method.
Since it feels like it takes me forever to write documentation, I'd rather not do it for code that is likely to be removed/changed in the near future.
But of course you're totally right. Javadoc would be useful here.